astrokat/observe_main.py

```diff
@@ -237,6 +238,7 @@ def __init__(self, opts, correlator=None):
 
     def __enter__(self):
         # Verify subarray setup correct for observation before doing any work
+        user_logger.warn('Running astrokat version - {}'.format(katversion.get_version()))
```

Comment:
In the screenshot from the CAM dry-run, the version doesn't look right.  It shows as `0.0...`.  We need to investigate this.  Probably to do with how it is installed, rather than this code, but we need to resolve it.

Reply:
OK, fixed logging output. CAM dry-run logging output is now correct.
Offline verify logging does not include branch number, but does include commit number, eg: 0.1.dev369+head.e851323.